### PR TITLE
Update UG to explain date display

### LINKED ISSUE #239: [PE-D] Inconsistency with display of date.
Some tasks have a day and date displayed while some only have the day. 

This is a slight issue as I will not be sure which date this Friday refers to. It would perhaps be better to just display day, date and time for all tasks so there is no confusion, especially since there is no way to sort by date in the application so tasks that have vastly differing due dates can all appear next to each other. 

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: Halpfrog/ped#14

### PR DESCRIPTION
**Pull request by snss231**
fixes #239

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -539,6 +539,21 @@ Imports a list of contacts from a .csv file.
 * `import fp/data/data.csv` will import all valid entries from the `data.csv` folder in the `/data` directory of the NUS Classes folder.
 
 
+
+### User-friendly date display
```
Comment (brian16600) [inline docs/UserGuide.md:543]: Should we put this at the top? Under Features or a separate section by itself

**Comment by snss231** [comment]
preview

File: docs/UserGuide.md (hunk 1 above)
Comment (snss231) [reply docs/UserGuide.md:543]: i'll leave it to you, currently it's at the bottom of "other features" which seems appropriate to me

Comment (brian16600) [reply docs/UserGuide.md:543]: Ok then, I think that does make sense

**Comment by junjunjieOng** [comment]
LGTM. Will merge it

**Pull request merged**